### ISSUE TITLE
Question about Normalization

### ISSUE THREAD

**Issue by sepidehkhakzad** [AlanBaade/MAE-AST-Public#4]
Hi! Great paper and Thanks for publishing the code!

I have a question about the normalization i.e. the following lines of code:

```
source = source.unsqueeze(1)
source = self.batch_norm(source)*0.5 # Mean 0, St dev 0.5
# Create image patches for masking via Unfold. BTC input shape BTC output shape
# Output continues to be unsqueezed from batch norm
source_patch = self.unfold(source).transpose(-1, -2)
```

I'm trying to run the model using a whole other dataset, but I've noticed that applying batch_norm in this way causes the target to change at every epoch and makes the target values smaller and smaller, so although the MSE loss decreases between the target and the prediction, this decrease doesn't mean a closer prediction to target, only that both have smaller values. Is this something you've noticed before? I was also wondering if you ever experienced consistent vertical lines in the prediction of this model throughout your training. Thank you.

**Comment by AlanBaade** [comment]
This behavior would happen if the batch norm parameters were learnable. I disabled learning batch norm parameters by setting affine=False ``self.batch_norm = nn.BatchNorm2d(num_features=1, affine=False)`` on line 209 of https://github.com/AlanBaade/MAE-AST-Public/blob/main/mae_ast/models/mae_ast.py

Is that part still the same in your codebase?

**Comment by sepidehkhakzad** [comment]
You are correct. That was my mistake and after changing it to False I no longer have the issue with target values becoming smaller at each epoch. Thank you.
